cinema/tests/test_movie_api.py

```diff
@@ -42,21 +40,15 @@
             self.assertEqual(movies.data[0][field], titanic[field])
 
     def test_get_movies_with_genres_filtering(self):
-        movies = self.client.get(
-            f"/api/cinema/movies/?genres={self.comedy.id}"
-        )
+        movies = self.client.get(f"/api/cinema/movies/?genres={self.comedy.id}")
         self.assertEqual(len(movies.data), 1)
-        movies = self.client.get(
-            f"/api/cinema/movies/?genres={self.comedy.id},2,3"
-        )
+        movies = self.client.get(f"/api/cinema/movies/?genres={self.comedy.id},2,3")
         self.assertEqual(len(movies.data), 1)
         movies = self.client.get("/api/cinema/movies/?genres=123213")
         self.assertEqual(len(movies.data), 0)
 
     def test_get_movies_with_actors_filtering(self):
-        movies = self.client.get(
-            f"/api/cinema/movies/?actors={self.actress.id}"
-        )
+        movies = self.client.get(f"/api/cinema/movies/?actors={self.actress.id}")
```

Comment:
In `test_get_movies_with_actors_filtering`, ensure that the actor IDs used in the query string are valid and exist in the database. This will help avoid false positives in the test results.